scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -195,37 +208,29 @@ class ThemeAggregator {
     const { theme, scoped } = options;
 
     if (!theme && !scoped) {
-      Logger.warn('No theme provided.', { context });
+      Logger.warn('No theme provided.', { LOG_CONTEXT });
       return false;
     }
 
     return true;
   }
 
   /**
-   * Generates a theme scss file and outputs it to the generated-themes directory.
-   * @param {object} contains requested scss file attrs
-   *   @param {array} assets - The aggregated theme files to import within generated file.
-   *   @param {string} themeName - Name of theme to aggregate.
-   *   @param {string} prefix - Prefix to append to generated file.
-   *   @param {string} scopeSelector - scss scope selector to encase theme.
-   *   @param {string} outputPath - path to write the scss file to. For testing purposes - overrides the default generated-themes path.
-   * @returns {string} - The path of the generated scss file, relative to the working home directory.
+   * Searches for theme files located within the following directory format: themes->theme-name->theme-name.scss.
+   * @param {string} themeName - The theme to aggregate.
+   * @param {Object} options - The aggregation options.
+   * @returns {array} - An array of ${themeName} files.
    */
-  static writeSCSSFile({
-    assets, themeName, prefix, scopeSelector, outputPath,
-  }) {
-    const fileName = `${prefix}-${themeName}.scss`;
-    const intro = `${DISCLAIMER}${scopeSelector}`;
-
-    let file = assets.reduce((acc, s) => `${acc}  @import '../${s}';\n`, '');
-    file = `${intro} {\n${file}}\n`;
+  static findThemeVariableFilesForGeneration(themeName, options = {}) {
```

Comment:
This looks one-to-one to the previous? Why the re-ordering?

Reply:
I mentioned this in the pull request description, but re-ordered the functions into a top-down execution format. I find this is easier to follow than the previous order